src/pages/ShareCodePage.tsx

```diff
@@ -67,6 +81,21 @@ function ShareCodePage({report}: ShareCodePageProps) {
     const platform = getPlatform();
     const isNative = platform === CONST.PLATFORM.IOS || platform === CONST.PLATFORM.ANDROID;
 
+    const logo = isReport ? getLogoForWorkspace(policy) : (UserUtils.getAvatarUrl(currentUserPersonalDetails?.avatar, currentUserPersonalDetails?.accountID) as ImageSourcePropType);
+
+    // default logos (avatars) are SVG and because of that require some special logic
+    let logoSVG: React.FC<SvgProps> | undefined;
+    let logoBackground;
+    let logoColor;
+
+    if (policy && !policy.avatar) {
+        logoSVG = ReportUtils.getDefaultWorkspaceAvatar(policy?.name) || Expensicons.FallbackAvatar;
+
+        const defaultWsAvatarColors = StyleUtils.getDefaultWorkspaceAvatarColor(policy?.name);
```

Comment:
I haven't seen `WS` abbreviation until this point, maybe let's not use it?
 ```suggestion
        const defaultWorkspaceAvatarColors = StyleUtils.getDefaultWorkspaceAvatarColor(policy?.name);
```